endpoints/openrtb2/auction.go

```diff
@@ -576,7 +576,15 @@ func (deps *endpointDeps) validateImp(imp *openrtb2.Imp, aliases map[string]stri
 	return nil
 }
 
-func validateBanner(banner *openrtb2.Banner, impIndex int) error {
+func isInterstitial(imp *openrtb2.Imp) bool {
+	if imp.Instl == 1 {
+		return true
+	}
+
+	return false
+}
```

Comment:
Nitpick: This method can be simplified to:
```
func isInterstitial(imp *openrtb2.Imp) bool {
	return imp.Instl == 1
}
```

Reply:
Changed. I must have had a typo somewhere because that was probably the first way I tried because that would work in Javascript :) Think I got this working though and hopefully cleaned!! Thanks @mansinahar and @SyntaxNode for the help on this!